src/extension/android/androidPlatform.ts

```diff
@@ -101,7 +102,7 @@ export class AndroidPlatform extends GeneralMobilePlatform {
                         .finally(() => {
                             return this.initializeTargetDevicesAndPackageName();
                         }).then(() => [this.debugTarget], reason => {
-                            if (reason.message === AndroidPlatform.MULTIPLE_DEVICES_ERROR && this.devices.length > 1 && this.debugTarget) {
+                            if (reason.message === ErrorHelper.getInternalError(InternalErrorCode.AndroidMoreThanOneDeviceOrEmulator).message && this.devices.length > 1 && this.debugTarget) {
```

Comment:
Can we simplify it to just use ErrorString class?

Reply:
We cannot do that because reason.message require strict match with the error string that will be rendered in our stdout console. This approach gives us such guarantees.